tests/test.wake.template

```diff
@@ -0,0 +1,16 @@
+global def exampleScalaModule = makeScalaModuleFromJSON here "example"
+
+global def runTests =
+  def paths = scalaModuleClasspath exampleScalaModule
+  def classpath = paths | map getPathName | catWith ":"
+  def cmd = which "java", "-cp", classpath, "example.Main", Nil
+  def visible = paths
+  def job = makePlan cmd visible | runJob
+  match job.getJobStatus
+    Exited i =
+      if i == 0 then
+        Pass "Passed"
+      else
+        Fail "Exited with status {i.format}".makeError
```

Comment:
This is a pretty minor nit, but Wake has literal extractors so you could write this as:
```
match job.getJobStatus
    Exited 0 = Pass "Passed"
    Excited i = Fail "Exited with status {i.format}".makeError
```

Reply:
Thanks for this feedback! This is one thing that I haven't trained myself to think about when pattern matching, and I very often only think in terms of the ADT types rather matching on specific instances of types.